plugins/search-backend/src/alpha.ts

```diff
@@ -97,8 +99,23 @@ export const searchPlugin = createBackendPlugin({
         permissions: coreServices.permissions,
         http: coreServices.httpRouter,
         searchIndexService: searchIndexServiceRef,
+        apiRegistryService: apiRegistryServiceRef,
+        pluginMetadataService: coreServices.pluginMetadata,
       },
-      async init({ config, logger, permissions, http, searchIndexService }) {
+      async init({
+        config,
+        logger,
+        permissions,
+        http,
+        searchIndexService,
+        apiRegistryService,
+        pluginMetadataService,
+      }) {
+        apiRegistryService.register(
```

Comment:
Just a question from an adopter perspective, would this be something we want to give adopters the ability to disable or optionally unregister for this or any other plugin via the `app-config.yaml` file?

Reply:
> My thinking was that adopters would want to see all installed plugins' APIs

A-ha, got it! That makes sense to me; thanks for clarifying 🙌🏻 !